src/_settings/mappings/career.json

```diff
@@ -0,0 +1,133 @@
+{
+  "properties": {
+    "active": {
+      "type": "boolean"
+    },
+    "applicant_types": {
+      "properties": {
+        "announcement_number": {
+          "type": "string"
+        },
+        "application_type": {
+          "properties": {
```

Comment:
So properties` is just something in the mapping that tells elasticsearch this is an object with the fields as it's members.